torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -1213,17 +1270,19 @@ void LoopNest::normalize(For* f, For** normalized) {
     throw malformed_input("normalize attempted on loop with no parent");
   }
 
-  if (!f->start()->isConstant()) {
+  if (startMustBeConstant && !f->start()->isConstant()) {
```

Comment:
@nickgg I might miss something here, I'm not sure why normalization cannot happen when the loop start is not a constant.

Reply:
I'm not sure either, @navahgar is there a strong reason for this restriction?

Reply:
Actually, there is no strong reason for this restriction. We were wondering if users would want this normalization, if the loop start is not a constant. That does not seem to be the case.

I do agree that we can have it on by default.

Reply:
@navahgar I removed the assumption that start must start with constant in https://github.com/pytorch/pytorch/pull/44133.